db/migrate/20190110135826_create_versioned_removal.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateVersionedRemoval < ActiveRecord::Migration[5.2]
+  def change
+    create_table :versioned_removals do |t|
+      t.string :explanatory_note
+      t.string :alternative_path
+      t.boolean :redirect, default: false
+
+      t.datetime :created_at, null: false
```

Comment:
Good catch. Will add.

Reply:
Since they have a one-to-one with a status we can use the status one. 

I don't think we have any other one-to-one relationships like this where both have a created_by